site/src/components/Resources/ResourceCard.stories.tsx

```diff
@@ -96,34 +69,38 @@ export const BunchOfMetadata: Story = {
         },
       ],
     },
-    agentRow: (agent) => (
-      <ProxyContext.Provider
-        value={{
-          proxyLatencies: MockProxyLatencies,
-          proxy: getPreferredProxy([], undefined),
-          proxies: [],
-          isLoading: false,
-          isFetched: true,
-          setProxy: () => {
-            return;
-          },
-          clearProxy: () => {
-            return;
-          },
-          refetchProxyLatencies: (): Date => {
-            return new Date();
-          },
-        }}
-      >
-        <AgentRow
-          showApps
-          key={agent.id}
-          agent={agent}
-          workspace={MockWorkspace}
-          serverVersion=""
-          onUpdateAgent={action("updateAgent")}
-        />
-      </ProxyContext.Provider>
-    ),
+    agentRow: getAgentRow,
   },
 };
+
+function getAgentRow(agent: WorkspaceAgent): JSX.Element {
```

Comment:
Why not create a component instead of having a `getAgentRow`? `ExampleAgentRow`?